File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.js
function solution(n, control) {
    var answer = n;
    for(let i of control){
        if(i==='w'){
            answer+=1
        }
        else if(i==='s'){
            answer-=1
        }
        else if(i==='d'){
            answer+=10
        }
        else{
            answer-=10
        }
    }
    return answer;
}

File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.java
class Solution {
    public int solution(int n, String control) {
        int answer = n;
        for(char s : control.toCharArray()){
            if(s=='w'){
                answer+=1;
            }
            else if(s=='s'){
                answer-=1;
            }
            else if(s=='d'){
                answer+=10;
            }
            else{
                answer-=10;
            }
        }
        return answer;
    }
}